src/ui/components/UserDefinesList/index.tsx

```diff
@@ -122,25 +140,37 @@ const UserDefinesList: FunctionComponent<UserDefinesListProps> = (props) => {
             {item.type === UserDefineKind.Text && item.enabled && (
               <>
                 <ListItem sx={styles.complimentaryItem}>
-                  {!item.sensitive && (
-                    <TextField
-                      size="small"
-                      onChange={onUserDefineValueChange(item.key)}
-                      value={item.value}
-                      fullWidth
-                      label={inputLabel(item.key)}
-                    />
-                  )}
-                  {item.sensitive && (
-                    <SensitiveTextField
-                      name={item.key}
-                      size="small"
-                      onChange={onUserDefineValueChange(item.key)}
-                      value={item.value}
+                  {item.key === UserDefineKey.BINDING_PHRASE && (
+                    <Autocomplete
+                      freeSolo
                       fullWidth
-                      label={inputLabel(item.key)}
+                      options={savedBindingPhraseHistory}
+                      renderInput={(params) => (
+                        <TextField {...params} label={inputLabel(item.key)} />
```

Comment:
Need to include all the properties that the other text field has, like value, onChange, size, etc.

For sensitive field to work with autocomplete, the sensitive field component needs to merge the InputProps it receives with the values it sets.  We probably only want to show the autocomplete when sensitive data is set to be shown so we don't end up in a situation where the current value is masked, but the historical values are shown in plain text.

![image](https://user-images.githubusercontent.com/38869875/160139062-9b31c1df-1bd7-4f1f-a173-e64701e1c9b6.png)